bin/game.rb

```diff
@@ -40,6 +47,13 @@ def execute
 
   private
 
+  def place_ai_mark
+    puts "board is: #{@board_gateway_hack.fetch_board.board}"
```

Comment:
why not just clone the board and use that in minimax instead?

Also, fetch_board.board could be renamed -  I'd expect fetch board to return the board without using .board